src/controllers/scripting/legacy/controllerscriptinterfacelegacy.h

```diff
@@ -17,6 +17,16 @@ class ConfigKey;
 class ControllerScriptInterfaceLegacy : public QObject {
     Q_OBJECT
   public:
+    enum class WellKnownCharsets {
+        LATIN_1,
+        ISO_8859_1,
+        LATIN_9,
+        ISO_8859_15,
+        UCS_2,
+        ISO_10646_UCS_2
```

Comment:
for `enum class` I prefer `CamelCase` (also looks nicer in JS). I don't know how to express the ISO names well, so I guess the current naming is fine.
```suggestion
        Latin1,
        ISO_8859_1,
        Latin9,
        ISO_8859_15,
        UCS2,
        ISO_10646_UCS_2
```